docs/docs/components/experimental.mdx

```diff
@@ -228,23 +182,10 @@ To use this component, provide the SQL query, specify the database URL, and opti
 
 This component dynamically generates a component from a flow. The output is a list of records with keys 'result' and 'message'.
```

Comment:
I'm not sure this is correct. This Component is more complex than this. Maybe we should write an example of what happens to it when used.